src/progress-steps/progress-steps.js

```diff
@@ -23,12 +23,17 @@ function ProgressSteps({
     if (!child) return;
 
     const childOverrides = child.props.overrides || {};
+
+    const isActive = child.props.isActive
```

Comment:
we may want to check using "typeof", so one can input false too

Reply:
yes, makes sense. fixed it in latest commit